server/api/team.js

```diff
@@ -31,41 +32,41 @@ router.post('team.users', auth(), pagination(), async ctx => {
   };
 });
 
-router.post('team.addAdmin', auth({ adminOnly: true }), async ctx => {
-  const { user } = ctx.body;
-  const admin = ctx.state.user;
-  ctx.assertPresent(user, 'id is required');
+router.post('team.addAdmin', auth(), async ctx => {
+  const userId = ctx.body.user;
+  const teamId = ctx.state.user.teamId;
+  ctx.assertPresent(userId, 'id is required');
 
-  const team = await Team.findById(admin.teamId);
-  const promotedUser = await User.findOne({
-    where: { id: user, teamId: admin.teamId },
-  });
-
-  if (!promotedUser) throw httpErrors.NotFound();
+  const user = await User.findById(userId);
+  authorize(ctx.state.user, 'promote', user);
 
-  await team.addAdmin(promotedUser);
+  const team = await Team.findById(teamId);
+  await team.addAdmin(user);
 
-  ctx.body = presentUser(ctx, promotedUser, { includeDetails: true });
+  ctx.body = {
```

Comment:
Updated the response here, this was missing in the user data with other parts of the response. These endpoints seem to be the only place where a `data` property wasn't used.